packages/ui/src/forum/components/PostList/PostListItem.tsx

```diff
@@ -13,55 +14,54 @@ import { spacing } from '@/common/utils/styles'
 import { ForumPost } from '@/forum/types'
 import { MemberInfo } from '@/memberships/components'
 
-import { BlockDate } from './BlockDate'
-
 interface PostProps {
   post: ForumPost
   isSelected?: boolean
 }
 
-export const ForumComment = forwardRef<HTMLDivElement, PostProps>(({ post, isSelected }, ref) => {
-  const { link, createdAtBlock, updatedAt, author, text, reaction, repliesTo } = post
+export const PostListItem = forwardRef<HTMLDivElement, PostProps>(({ post, isSelected }, ref) => {
+  const { createdAtBlock, updatedAt, author, text, reaction, repliesTo } = post
   const edited = useMemo(() => updatedAt && <EditionTime>(edited {relativeTime(updatedAt)})</EditionTime>, [updatedAt])
 
   return (
-    <ForumCommentStyles ref={ref} isSelected={isSelected}>
-      {author && <MemberInfo member={author} />}
-      <BlockDate block={createdAtBlock} />
-
+    <ForumPostStyles ref={ref} isSelected={isSelected}>
+      <ForumPostRow>
+        <ForumPostAuthor>{author && <MemberInfo member={author} />}</ForumPostAuthor>
+        <BlockTime block={createdAtBlock} layout="reverse" />
+      </ForumPostRow>
       <MessageBody>
         {repliesTo && (
           <Reply>
             <ReplyBadge>
               <ArrowReplyIcon />{' '}
               <Badge>
-                <Link to={repliesTo.link}>Replies to {repliesTo?.author?.handle}</Link>
+                <Link to={window.location.href}>Replies to {repliesTo?.author?.handle}</Link>
```

Comment:
FYI: This will link to the whole thread but it should link to the specific post. I know that this will be handled in another issue so for now it is OK.